modules/tinymce/src/plugins/table/main/ts/ui/MenuItems.ts

```diff
@@ -244,6 +244,29 @@ const addMenuItems = (editor: Editor, selectionTargets: SelectionTargets, clipbo
     }),
     onSetup: selectionTargets.onSetupCellOrRow
   });
+
+  const tableCellBorderWidthsList = getTableBorderWidths(editor);
+  editor.ui.registry.addNestedMenuItem('tablecellvalign', {
+    icon: 'vertical-align',
+    getSubmenuItems: () => generateItems(editor, tableCellBorderWidthsList, 'tablecellborderwidth', (item) => item.title, (item) => {
+      editor.execCommand('mceTableApplyCellStyle', false, {
+        'border-width': item.value
+      });
```

Comment:
This is now being duplicated 6 times with just a different value (3 in each file). Can you please abstract that out into a helper function?

- [ ] Clean up duplication

Reply:
We're likely going to need using a proper onAction here later, and I'd prefer not to have to double the amount of subentry-generators when we do, so I'd prefer to keep this onAction-functionality.

Reply:
That's not what I meant sorry, I literally meant a function to do the exec command. eg:
```
const applyTableCellStyleAction = (style: string) => (item: Item) => {
  editor.execCommand('mceTableApplyCellStyle', false, { [style]: item.value });
};
```
and then that could be used like so:
```
generateItems(editor, tableCellBorderWidthsList, 'tablecellborderwidth', applyTableCellStyleAction('broder-width'))
```